massdriver-application-gcp-cloud-function/bucket.tf

```diff
@@ -0,0 +1,18 @@
+resource "google_storage_bucket" "main" {
```

Comment:
Cloud Functions needs source code in order to provision. For the first provision and subsequent ones with user code, this bucket will be the storage destination. Kinda like a container repository for zip files.